priv/repo/migrations/20191127112248_remove_people_key_id.exs

```diff
@@ -0,0 +1,9 @@
+defmodule App.Repo.Migrations.RemovePeopleKeyId do
+  use Ecto.Migration
+
+  def change do
+    alter table("people") do
+      remove :key_id
```

Comment:
@SimonLab **`:key_id`** is for storing the id of the encryption key used to encrypt the data in the same _row_ as the data so that it's clear which key to use to _decrypt_ it.
Are you planning to store the encryption key id somewhere `else`? 💭

Reply:
Yes the key_id has been added to the `sessions` table
see https://github.com/dwyl/app/issues/247#issuecomment-557055588

Reply:
@SimonLab yeah, I saw that 👍 
_However_ I expect the data in these two tables will be encrypted with _different_ encryption keys.
If someone **registers** to use the App using just an email address and password https://github.com/dwyl/app/issues/237 
(_a use-case we will 100% be adding to support people who don't have/use Google Accounts_) 
Then we will need to have **`key_id`** in the **`people`** table to know which encryption key was used to encrypt their data on **sign-up**.
On each subsequent **login** the ***session*** data (_e.g: IP Address and Browser User Agent_) will be encrypted and stored in the **`sessions`** table _possibly_ with a _different_ encryption key because of our key rotation system. 💭

Reply:
Yes makes sense, sorry I was focused on the google auth and hasn't thought of keeping the specific key_id for email/password authentication. I'll add it back :+1:

Reply:
Glad with the _new_ solution for this. in-lining the `key_id` is more "user-friendly" (_to devs_) 👍